src/Agent.Listener/Configuration/CredentialManager.cs

```diff
@@ -18,7 +18,8 @@ public class CredentialManager : AgentService, ICredentialManager
         public static readonly Dictionary<string, Type> CredentialTypes = new Dictionary<string, Type>(StringComparer.OrdinalIgnoreCase)
         {
             { "PAT", typeof(PersonalAccessToken)},
-            { "ALT", typeof(AlternateCredential)}
+            { "ALT", typeof(AlternateCredential)},
```

Comment:
Is this Basic? If so we should probably use the work Basic instead. "Alternate credentials" is a thing today with the hosted service, but I believe we typically advocate PAT instead and discourage Alternate Credentials for hosted.

Also, the existing Windows agent supports "Basic" today: https://msdn.microsoft.com/en-us/library/vs/alm/build/agents/windows